guides/common/modules/ref_system-requirements.adoc

```diff
@@ -94,6 +94,16 @@ ifdef::foreman-el,katello,satellite[]
 SELinux must be enabled, either in enforcing or permissive mode.
 Installation with disabled SELinux is not supported.
 
+.Synchronized system clock
+The system clock on the base operating system where you are installing your {ProductName} must be synchronized with Network Time Protocol (NTP) servers.
```

Comment:
*technically* there is no requirement that the synchronization happens via NTP (and other protocols exist and are being used).

Reply:
Addressed.